src/cc_catalog_airflow/dags/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -0,0 +1,177 @@
+    query_param,
+    endpoint = ENDPOINT,
+    retires = RETRIES
+    ):
+
+    if retires < 0:
+        logger.error('Max retries exceeded. Failure')
+        raise Exception('retries exceeded')
+        return None
+
+    response = delay_request.get(
+                endpoint,
+                query_param
+                )
+
+    if response.status_code == 200 and response is not None:
+        logger.info('Response status 200')
+        try:
+            response_json = response.json()
+            return response_json
+        except:
+            logger.warning('response not captured')
+            raise Exception('json response could not be captured')
+            response_json = None
+
+    if (
+        response_json is None or response_json.get('error') is not None
+        ):
+        logger.warning('Bad response_json')
+        logger.info('Retrying \n'
+                    f'endpoint -- {ENDPOINT} \t'
+                    f' with parameters -- {query_param} ')
+
+        _get_response(
+            query_param,
+            endpoint,
+            retries = retries - 1
+            )
+    
+def _response_handler(
+    batch
+    ):
+    for data in batch:
+        license_ = data.get('share_license_status','').lower()
+        
+        if license_ != 'cc0':
+            logger.error('Wrong license image')
+            continue
+        license_version = '1.0'
+
+        foreign_id = data.get('id')
+        foreign_landing_url = data.get('url',None)
+        image_data = data.get('images',None)
+        image_url,key = _get_image_type(image_data)
+
+        if image_url is not None:
+            width = image_data[key]['width']
+            height = image_data[key]['height']
+        
+        title = data.get('title',None)
+        metadata = _get_metadata(data)
+        
+        if data.get('creators'):
+            creator_name = data.get('creators')[0].get('description','')
+        else:
+            creator_name = ''
+
+        total_images = image_store.add_item(
+                        foreign_landing_url=foreign_landing_url,
+                        image_url=image_url,
+                        license_=license_,
+                        license_version=license_version,
+                        foreign_identifier=foreign_id,
+                        width=width,
+                        height=height,
+                        title=title,
+                        creator=creator_name,
+                        meta_data=metadata,
+                        source=PROVIDER
+                        )
+    
+    return total_images
+
+def _get_image_type(
+    image_data
+    ):
+    if image_data.get('web'):
+        key = 'web'
+        image_url = image_data.get('web').get('url',None)
+    
+    elif image_data.get('print'):
+        key = 'print'
+        image_url = image_data.get('print').get('url',None)
+
+    elif image_data.get('full'):
+        key = 'full'
+        image_url = image_data.get('full').get('url',None)
```

Comment:
These `get` calls break whenever `image_data` is `None`.  This prevents the script from running.